src/datatype.rs

```diff
@@ -181,7 +180,7 @@ impl DataType {
                 let secs = days * 86400;
                 chrono::NaiveDateTime::from_timestamp_opt(secs, 0)
             }
-            DataType::Float(f) | DataType::DateTime(f) => {
+            DataType::Float(f) => {
```

Comment:
I think, it (Float/Int conversation) isn't necessary and can be removed in the future.

Reply:
Not sure we can discard it. I wouldn't trust the excel file to have it properly encoded every time and I'd expect people to try getting a datetime anyway.
On the other maybe it is better to create an `ExcelDateTime` and call `as_datetime`?

Reply:
> On the other maybe it is better to create an ExcelDateTime and call as_datetime?

Thanks, I dit it.